src/microbe_stage/Microbe.cs

```diff
@@ -2108,6 +2110,137 @@ private Transform GetNewPhysicsRotation(Transform transform)
         return new Transform(new Basis(slerped), transform.origin);
     }
 
+    /// <summary>
+    ///   Adds torque to reach the targeted rotation
+    /// </summary>
+    private void ApplyPhysicsRotation(PhysicsDirectBodyState physicsState)
+    {
+        var transform = physicsState.Transform;
+        var target = transform.LookingAt(LookAtPoint, new Vector3(0, 1, 0));
+        if (transform == target)
+            return;
+
+        // Strength can later be affected by other values
+        // More strength is required to rotate larger microbes
+        var strength = Constants.CELL_BASE_THRUST * (1.0f + HexCount * 0.8f);
+
+        // Calculate the distance to reach the target rotation in degrees
+        var distance = target.basis.GetEuler().y - transform.basis.GetEuler().y;
+        distance = (float)(distance * (180.0f / Math.PI));
+        if (distance > 180.0f)
+        {
+            distance = -360.0f + distance;
+        }
+        else if (distance < -180.0f)
+        {
+            distance = 360.0f + distance;
+        }
+
+        var force = 0.0f;
+
+        // Angular velocity of the microbe
+        var velocity = physicsState.AngularVelocity.y;
+
+        // Maximum desired angular velocity
+        var maxVelocity = 5.0f;
+
+        // Brake if velocity exceeds maxVelocity
+        if (velocity < -maxVelocity)
+        {
+            force = 1.0f;
+        }
+        else if (velocity > maxVelocity)
+        {
+            force = -1.0f;
+        }
```

Comment:
Here you should be able to make it much smaller using Mathf.Sign() to get the sign of your numbers, as well as Mathf.Abs()